src/index.html

```diff
@@ -1,5 +1,8 @@
 <!doctype html>
-<html lang="en">
+<html
+  class="page"
+  lang="en"
+>
   <head>
     <meta charset="UTF-8" />
```

Comment:
The `<meta>` tag should be indented by 2 spaces within the `<head>` tag to maintain consistent indentation.